saleor/product/migrations/0142_auto_20210201_1459.py

```diff
@@ -0,0 +1,160 @@
+# Generated by Django 3.1.5 on 2021-02-01 14:59
```

Comment:
Could you create migration with custom name? We trying to avoid auto generated name for data migrations.